### DISCUSSION TITLE
Does TIC-80 support uploading a picture and designing based on that?

### DISCUSSION THREAD

**Discussion by l-cli** [nesbox/TIC-80#1330] [Q&A]
It would be great if we can upload something (like a template, a rough map, etc.) and then build on that, like draw maps on that given picture - saves a lot of time compared to copy a map by brute force!

**Comment by RobLoach** [comment]
You use to be able to load your own sprite-sheet.

```
tic80 mycart.tic -sprites myimage.gif
```

**Comment by joshgoebel** [comment]
Can't you do this with `import` as well?
